xml/deploy_caasp.xml

```diff
@@ -10,11 +10,12 @@
  xml:id="deploy-rook">
 <!-- ============================================================== -->
  <title>Deploying on Top of &casp; Cluster</title>
+
  <warning>
   <title>Limited Availability</title>
   <para>
-   Running containerized &ceph; cluster on &casp; is a technology preview. Do
-   not deploy on a production &kube; cluster.
+   Running containerized &ceph; cluster on &casp; is released with limited
+   availability for the &productname; &productnumber; release.
```

Comment:
Sounds a bit like "This release is released with limited release in the release."

I think this could be better if rephrased; agreed?